apps/heft/src/cli/actions/RunAction.ts

```diff
@@ -78,21 +78,18 @@ export function definePhaseScopingParameters(action: IHeftAction): IScopingParam
   return {
     toParameter: action.defineStringListParameter({
       parameterLongName: Constants.toParameterLongName,
-      parameterShortName: Constants.toParameterShortName,
```

Comment:
Yeah. I spoke with @octogonz and given that there are only 52 short-name variants (a-z, A-Z), we should probably leave them to be available to plugins. One example actually came up in this PR, where the `-t`/`--to` parameter was preventing the `-t`/`--test-path-pattern` parameter from the Heft Jest plugin.

Reply:
To be fair, passing the scope selectors to Heft is a bit of a rare case and unlikely to be used by typical developers. Though also, don't we have the `--` split to separate arguments for the runner from arguments for the action?

Reply:
In general, short names should be introduced when there is an obvious pressing need, because a certain parameter is heavily used.  Adding short names "just in case" causes lots of trouble, due to complex interactions of expectations/consistence.

In this example `--to`  `--from` etc correspond to Rush selectors. Thus everyone will reasonably expect the short names to correspond to Rush's short names.  But in Heft these parameters must coexist with other parameters, that have other expectations. It's a bit of a sudoku puzzle to end up with a set of intuitive parameter names that avoid all such conflicts. In other words, let's avoid making that puzzle harder than it needs to be. 😄